desktop section revamp (#39)
cryptomator/docs: source/desktop/vault-management.rst

Vault Management
================

A *vault* is where your files are stored encrypted.

> For your operating system or other apps a vault is a just a normal directory containing some encrypted files.

Reviewer: ```suggestion
For your operating system or other apps, a vault is a just a normal directory containing some encrypted files.
```

> Only Cryptomator understands can decrypt the vault's contents when you unlock it using a password.

Reviewer: ```suggestion
Only Cryptomator can decrypt the vault's contents when you unlock it using a password.
```

.. _desktop/vault-management/remove-vaults:

Remove Vaults
-------------

To remove a vault from the vault list, right click on a vault, and click remove. 
This is only possible if the vault is locked.

.. note::

>     The vault's is **not** deleted from your PC by removing it from the list. You need to delete the vault directory to permentantly delete your encrypted files using the file manager.

Reviewer: ```suggestion
    The vault is **not** deleted from your PC by removing it from the list. If you wish to permanently delete your encrypted files, you need to delete the vault directory using the file manager.
```

Vault Options
-------------

> Each vault has it's own settings which can be customized under vault options.

Reviewer: ```suggestion
Each vault has its own settings which can be customized under vault options.
```

> To open a vault's settings, select a vault, lock it, and click on ``Vault Options`` (gear icon).

Reviewer: ```suggestion
To open a vault's settings, select a vault, lock it, and click on ``Vault Options``.
```

Maybe we shouldn't mention the gear icon for "vault options" since we use a similar icon for the preferences.

The options are divided across three categories:

> 1. General - Options not fitting in other categories

Reviewer: ```suggestion
1. General - Options not fitting in other categories.
```
